.eslintrc.json

```diff
@@ -35,6 +44,7 @@
     "@typescript-eslint/no-explicit-any": "off",
     "@typescript-eslint/no-unused-expressions": "error",
     "react/display-name": "off",
+    "react/function-component-definition": "off",
```

Comment:
this only removes type defn from react components, right?
 normal functions:
```
function concat(a: numb, b: num): string { return `${a}${b}` }
```
will have return type defn, correct?